pkg/investigations/cpd/cpd.go

```diff
@@ -65,8 +67,8 @@ func InvestigateTriggered(r *investigation.Resources) error {
 
 	// Check if the OCM Error code is a known error
 	if len(r.Cluster.Status().ProvisionErrorCode()) > 0 && r.Cluster.Status().ProvisionErrorCode() != unknownProvisionCode {
```

Comment:
This does mean that it is a known error, but does not necessarily mean that SRE does not need to investigate it/that a SL has been sent. There some relatively generic errors which we define so that we can classify our CPDs, e.g. `GeneralOperatorDegraded` is classified as a known error code, but does not send a service log. Every CPD reaching PagerDuty should generally imply that an investigation is needed.

Reply:
Ah, I see. I'll change the PR to instead make it clearer that SRE has to investigate anyway. Thanks for that key information!